.github/dependabot.yml

```diff
@@ -0,0 +1,11 @@
+# .github/dependabot.yml
```

Comment:
Can we configure dependabot so it provides all updates to dependencies in a single PR every month?

See:
- https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#groups
- https://docs.github.com/en/code-security/dependabot/working-with-dependabot/troubleshooting-dependabot-errors#dependabot-fails-to-group-a-set-of-dependencies-into-a-single-pull-request-for-dependabot-version-updates